src/pages/Results.pug

```diff
@@ -50,28 +50,62 @@ block article
         Considering the slight curvature, all samples have an estimated length of around 250 nt.
         This suggests no effect of initial nucleotide concentration in the tested range.
         It must be noted that this is only true for an incubation time of 10 min.
-        To fully understand the effect of the initial nucleotide concentration, this experiment should be adjusted, and a time-dependent reaction should be performed for every concentration tested since [3] claim that the initial nucleotide concentration is a reliable way of tweaking the reaction speed. 
-        The results presented here are only a snapshot, which cannot serve to make reliable suggestions.
+        To fully understand the effect of the initial nucleotide concentration, this experiment should be adjusted, and a time-dependent reaction should be performed for every concentration tested.
         However, we chose to continue to work with our standard concentration of 100 µM as they were sufficient for the characterizations.
         Nevertheless, this concentration may be subject to change depending on the experiment performed.
         
         ## Varying Primer Concentration
+        
         ## Varying Cofactor Concentration
+        One attempt to tweak the reaction speed was to adjust the cofactor availability by reducing its concentration.
+        Since the incorporation speed of nucleotide T by far exceeds that of all other nucleotides, we performed TdT reactions with varying concentrations of the TdT reaction buffer containing the cofactor.
+        As the cofactor was inherently present in the buffer, we had to adjust the buffer concentration in total and could not just alter the cofactor concentration itself, which might have unforeseen effects.
+        
+    - var imgpath = "" + require("../assets/img/Results/2021-06-24_01.png")["default"]
+    +image(2, imgpath, "Figure 3 TdT tailing reaction of an AT-rich ssDNA primer with dTTP and various cofactor concentrations at RT (24 °C) for 30 min. Cofactor was inherently present in reaction buffer, so that buffer concentration was consequently adjusted as well. 2.5% Agarose with TBE, SYBR Gold staining, 90 V, 60 min. M: GeneRuler 50 bp DNA Ladder 1: AT.rich primer reference 2: 2 mM CoCl2 3: 1.5 mM CoCl2 4: 1.0 mM CoCl2 5: 0.5 mM CoCl2 6: 0.0 mM CoCl2 negative control.")
```

Comment:
```suggestion
    +image(3, imgpath, "Figure 3 TdT tailing reaction of an AT-rich ssDNA primer with dTTP and various cofactor concentrations at RT (24 °C) for 30 min. Cofactor was inherently present in reaction buffer, so that buffer concentration was consequently adjusted as well. 2.5% Agarose with TBE, SYBR Gold staining, 90 V, 60 min. M: GeneRuler 50 bp DNA Ladder 1: AT.rich primer reference 2: 2 mM CoCl2 3: 1.5 mM CoCl2 4: 1.0 mM CoCl2 5: 0.5 mM CoCl2 6: 0.0 mM CoCl2 negative control.")
```